docs/dev_guide/tests.rst

```diff
@@ -248,28 +248,18 @@ Here is a simple example: ::
 The current figure at the end of the unit test, or an explicitly returned figure, has its hash compared against an established hash library (more on this below).
```

Comment:
We should also mention that the hashing algorithm used is `SHA256` and the hash is stored as a `HEX` string.

Reply:
I added the algorithm.